src/NpmUtilities.js

```diff
@@ -30,6 +30,25 @@ export default class NpmUtilities {
   }
 
   @logger.logifySync()
+  static getWhoIAm() {
+    try {
+      return ChildProcessUtilities.execSync("npm whoami", { stdio: "none" });
+    } catch (e) {
+      return undefined;
+    }
+  }
+
+  @logger.logifySync()
+  static getConfig(key) {
+    return ChildProcessUtilities.execSync(`npm config get ${key}`);
+  }
+
+  @logger.logifySync()
+  static setConfig(key, value) {
```

Comment:
Mostly symmetry, but also because [the inital comment](https://github.com/lerna/lerna/issues/55#issue-133396688) explicitly mentioned "logg[ing] you back out afterwards."